Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -24,26 +24,29 @@
 % Build a model of a one-leg hopper, with one muscle.
 
 p = inputParser();
-defaultMuscleModel = 'Thelen2003';
+defaultMuscleModel = 'Millard2012Equilibrium';
 defaultMaxIsometricForce = 4000.0;
 defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
-defaultActivation = [0.0 1.99 2.0 3.89 3.9 4.0;
+defaultExcitation = [0.0 1.99 2.0 3.89 3.9 4.0;
                      0.3 0.3  1.0 1.0  0.1 0.1];
-defaultPrintModel = false; 
+defaultPrintModel = false;
+defaultAdditionalMass = 0;
```

Comment:
In a separate PR, I'm going to add another function called InteractiveHopperDefaults which gets called at the start up of the GUI and whenever the user clicks the 'Reset to Defaults' button I've added. That would definitely be able to be left within InteractiveHopper, but, if it's nice to make it a separate file, we could do that too.